docs/guides/platform/Install_MERLIN_In_AirGap.md

```diff
@@ -2,93 +2,91 @@
 
 IBM i Modernization Engine for Lifecycle Integration can be installed on a RedHat OpenShift Container Platform in that has no internet connectivity.
 
-   * Prerequisites
-   * Prepare a bastion host
-   * Prepare a local Docker registry
-       * Prepare the multi-architecture registry
-       * Configure the registry
-   * Prepare to install IBM i Modernization Engine for Lifecycle Integration
-       * Create environment variables for the installer and image inventory
-       * Download the CASE installer
-       * Log in to the OpenShift Container Platform as a cluster administrator
-       * Create a Kubernetes namespace for IBM i Modernization Engine for Lifecycle Integration
-       * Obtain the entitlement key
-       * Mirror the images and configure the cluster
-       * Create the IBM i Modernization Engine for Lifecycle Integration catalog source
-   * Install IBM i Modernization Engine for Lifecycle Integration
-       * Install by using the CLI
-       * Install by using the OpenShift Container Platform console
-
 ## Prerequisites
 
 An OpenShift Container Platform cluster must be installed and meet the resource requirements. For the supported OpenShift Container Platform versions and requirements, see [Supported product versions](./guides/platform/Install_IBM_i_Modernization_Engine_for_Lifecycle_Integration.md).
 
-A bastion server must be configured. For more information, see [Prepare a bastion host](#bastionhost).
+A bastion server must be configured. For more information, see [Prepare a bastion host](#prepare-a-bastion-host).
 
-A local Docker registry that is accessible from both the bastion server and the OpenShift Container Platform cluster nodes must be available. For more information, see [Prepare a Docker registry](#dockerregistry).
+A local Docker registry that is accessible from both the bastion server and the OpenShift Container Platform cluster nodes must be available. For more information, see [Prepare a Docker registry](#prepare-a-local-docker-registry).
 
-## Prepare a bastion host 
+## Prepare a bastion host
 
 Prepare a bastion host that can access the OpenShift Container Platform cluster, the local Docker registry, and the internet. The bastion host must be on a Linux® platform with any operating system that the IBM Cloud Pak® CLI and the OpenShift Container Platform CLI support.
 
 Complete these steps on the bastion node:
-* Install Docker or Podman.
-To install Docker, run these commands:
-```
+
+* Install Docker or Podman
+  To install Docker, run these commands:
+
+```bash
 yum check-update
 yum install docker
 ```
-Start the Docker service.
-```
+
+Start the Docker service
```

Comment:
Is this supposed to have a `*` to make it a bullet? There are a few other steps below this on the same page missing the bullet.

Reply:
This is part of `Install Docker or Podman` which is already a bullet. I changed the indentation a bit to make it look better.

Reply:
@SanjulaGanepola Changes made in 6d0afe6